jisantuc/AFDG
# Add base actions

Overview
-----

This PR adds types, messages, and interactions for adding bases to and removing
bases from tiles.

Testing
-----

No format tests yet

- add some units to some of the existing tiles from when you load the app
- remove bases from those places
- try to add bases back and notice that it fails for one of the players
- click "Switch players"
- Note that you can add bases for the other user now

Closes #19
